kyo-kernel/shared/src/main/scala/kyo/Kyo.scala

```diff
@@ -38,47 +43,41 @@ object Kyo:
     inline def unit[S]: Unit < S = ()
 
     /** Zips two effects into a tuple.
-      *
```

Comment:
some cleanup to avoid the file getting too large with the new versions with up to 8 elements